### PR TITLE
[Ready] Got some pickups working

### PR DESCRIPTION
**Pull request by husho**
- [x] Health pickups give health
- [x] Armour pickups give armour
- [x] Hidden packages can be collected and give money
- [x] Script pickups that used in missions can be collected
- [x] Adrenaline pills
- [x] Set x2 player walk start animation speed on pills pickup (seems like walk start animation is not implemented anyway)
- [x] ~~~Fix Time scale for pills~~~ in different PR 
- [x] ~~~Police bribes (requires wanted system)~~~ not going to implement without wanted system
- [x] Money (not used anywhere yet, except marked man)
- [x] Fix rampage can be only picked up on mission = 0
- [x] Dont render rampage pickups on mission = 1
- [x] Display text when pickup hidden packages
- [x] Opcodes 
- [x] ~~~Colours are incorrect~~~
- [x] Add new pickups to debug playground
- [x] ~~Adrenaline pill misaligned~~~ like in original game
- [x] ~~~A Drop In The Ocean packages~~~ (without physics for now)

### CODE REVIEW

**Comment by dracc** [comment]
You're on fire! Nice work! :D

File: rwengine/src/objects/PickupObject.cpp [hunk 1]
```diff
@@ -225,3 +227,142 @@ void PickupObject::setEnabled(bool enabled) {
...
+        // @todo display "heres little something"
+    } else {
+        state->playerInfo.hiddenPackagesCollected++;
+        state->playerInfo.money += 1000;
+
+        // @todo display "hidden packages found x of y"
+        const auto gxtEntry = "CO_ONE";
+
+        auto str = ScreenText::format(
+            character->engine->data->texts.text(gxtEntry),
```
Comment (danhedron) [inline rwengine/src/objects/PickupObject.cpp]: There should be an engine member here already, without needing to access through the player.

File: rwgame/RWGame.cpp [hunk 2]
```diff
@@ -414,26 +415,28 @@ int RWGame::run() {
...
+        if (!world->isPaused()) {
             accumulatedTime += deltaTime;
-            world->dynamicsWorld->stepSimulation(deltaTime * timescale, kMaxPhysicsSubSteps, kPhysicsTimeStep);
+            world->dynamicsWorld->stepSimulation(deltaTime, kMaxPhysicsSubSteps,
+                                                 deltaTime);
         }
 
         RW_PROFILE_BEGIN("Update");
-        while (accumulatedTime >= GAME_TIMESTEP && !world->isPaused()) {
+        while (accumulatedTime >= deltaTime && !world->isPaused()) {
```
Comment (ShFil119) [inline rwgame/RWGame.cpp]: Still do you think there's need with tickrate 60Hz?

File: rwengine/src/objects/PickupObject.hpp [hunk 3]
```diff
@@ -85,4 +93,108 @@ class PickupObject : public GameObject {
...
+
+/**
+ * @brief The RampagePickup class
+ */
+class RampagePickup : public PickupObject {
+public:
+    RampagePickup(GameWorld* world, const glm::vec3& position,
+                  BaseModelInfo* modelinfo, PickupType type);
+
+    bool isRampage() {
```
Comment (ShFil119) [inline rwengine/src/objects/PickupObject.hpp]: override

File: rwgame/RWGame.cpp (hunk 2 above)
Comment (husho) [reply rwgame/RWGame.cpp]: I'll open another PR for this.

**Pull request merged**